include/boost/geometry/extensions/contrib/ttmath_stub.hpp

```diff
@@ -103,6 +107,12 @@ namespace ttmath
         return two * ATan((sqrt(x * x + y * y) - x) / y);
     }
 
+    // needed in order to work with boost::geometry::math::fmod
+    template <uint Exponent, uint Mantissa>
+    inline Big<Exponent, Mantissa> fmod(Big<Exponent, Mantissa> const& x, Big<Exponent, Mantissa> const& y)
```

Comment:
I am aware of that.
I was waiting to have the okay for the other PRs before modifying this.
I was also considering closing this one and creating a new one so as to keep the history simple.
Will re-act on it once the other PRs are okay/done.